source/documentation/deploying_services/index.md

```diff
@@ -650,13 +650,235 @@ Note that data restore will not be available in the event of an RDS outage that
 
 For more details about how the RDS backup system works, see [Amazon's DB Instance Backups documentation](http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.BackingUpAndRestoringAmazonRDSInstances.html) [external page].
 
-## MongoDB
+## Redis
 
-MongoDB is an open source cross-platform document-oriented database program. It uses JSON-like documents with schemas, and is often used for content management such as articles on [GOV.UK](https://www.gov.uk/). This is an early version of the service that is available on request so that we can get feedback, and we will make you aware of any constraints in its use at that time.
+Redis is an open source in-memory data store that can be used as a database cache or message broker. This is a public beta version of the service that is available on request so that we can get feedback. We will make you aware of any constraints in its use at time of request.
 
-## Redis
+### Set up a Redis service
+
+To set up a Redis service:
+
+1. Run the following code in the command line to see what plans are available for Redis:
+
+    ```
+    cf marketplace -s redis
+    ```
+
+    There is currently only one plan available for Redis:
+
+    ```
+    service plan   description                                    free or paid
+    tiny           568MB RAM, 1 shard, single node, no failover   free
+    ```
+
+1. Run the following code to create a service instance:
+
+    ```
+    cf create-service redis PLAN SERVICE_NAME
+    ```
+
+    where `PLAN` is the plan you want, and `SERVICE_NAME` is a unique descriptive name for this instance of the service. For example:
+
+    ```
+    cf create-service redis tiny my-redis-service
+    ```
+
+1. It will take between 5 and 10 minutes to set up the service instance. To check its progress, run:
+
+    ```
+    cf service SERVICE_NAME
+    ```
+
+    for example:
+
+    ```
+    cf service my-redis-service
+    ```
+
+    The service instance is set up when the `cf service SERVICE_NAME` command returns a `create succeeded` status. Here is an example of the output you will see:
+
+    ```
+    name:            my-redis-service
+    service:         redis
+    bound apps:      
+    tags:            
+    plan:            tiny
+    description:     AWS ElastiCache Redis service
+    documentation:   
+    dashboard:       
+
+    Showing status of last operation from service my-redis-service...
+
+    status:    create succeeded
+    message:   ---
+               status               : available
+               cluster id           : cf-u7zpvbwzxmrvu
+
```

Comment:
The weird formatting in this output (the extra newlines / misaligned values) are due to a quirk in how the `cf` command .... I don't think we should replicate the quirk in this example output since it makes it harder to read.